### DISCUSSION TITLE
Dashboard / Drag and drop sections

### DISCUSSION THREAD

**Discussion by DeeZeeDeez** [mainsail-crew/mainsail#1823] [General]
Drag and drop sections on the dashboard page.
Thank you for everything and keep up the good work!

**Comment by meteyou** [comment]
What do you mean with "Drag and drop sections on the dashboard"? Can you please describe this a little bit?

**Comment by DeeZeeDeez** [comment]
@meteyou

On mobile, the Webcam section is always at the bottom.
Would be nice to click and hold the Webcam section and move it around and reorder it as we wish. Let say first position at the top.

Same on my PC. Take a look at my screen shots. 
Would be nice to drag the Miscellaneous, BL Touch and Calibration section in the second row so I can see every section while full screen at the top of the page.

![Screen](https://github.com/mainsail-crew/mainsail/assets/112959094/c3868b80-1d5c-4190-b2dd-e5c8eba3a8c0)
![Screen2](https://github.com/mainsail-crew/mainsail/assets/112959094/fb579219-8a27-4146-8641-366570ba346d)

It's just an idea...
That would add another way of customization for the end user.

**Comment by meteyou** [comment]
just open the "interface settings" (the cogs icon on the right top from the interface), go into the "dashboard" tab and move all panels. you can also sort it on different viewport and enable/disable panels.

**Comment by DeeZeeDeez** [comment]
@meteyou Wooooww, didn't know we could re-organise from this place. Thank youuuu !!
